salt/metalk8s/volumes/prepared/files/metalk8s-volume-clean-up

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+from __future__ import print_function
+import os
+import subprocess
+import sys
+
+
+def die(message):
+    print(message, file=sys.stderr)
+    sys.exit(1)
+
+
+def get_volume_id():
+    if len(sys.argv) != 2:
+        die("Invalid number of arguments: expected only one volume ID")
+
+    return sys.argv[1]
+
+
+def find_device(volume_id):
+    path_by_uuid = "/dev/disk/by-uuid/{}".format(volume_id)
+    if os.path.exists(path_by_uuid):
```

Comment:
Could even use `os.path.islink` (or Pathlib, but since it's Python 2, not really).

Reply:
It's a link because that's how `udev` generally sets things up. However, it *could* (theoretically) be a proper block device. Why rule that out?

Reply:
In that case, better keep the `os.path.exists` yeah.

Reply:
In metalk8s-2.7 we will likely go with Salt Python3 so we could likely assume we run with Python3

Reply:
Kept it.